download/index.shtml

```diff
@@ -89,27 +87,11 @@
     <dl>
     <dt class="im"><img src="../images/ico-jmri.gif" width="34" height="34" alt="JMRI logo"></dt>
     <dd>
-	<a id="prod-download"></a><p class="dl">Download:
```

Comment:
I suggest we leave out this change until we are on master again, so that @bobjacobsen (mentioning does not seem to work here) can confirm there is no release build script trying to put stuff here.
Happy to copy all your other suggestions to branch.

Reply:
I'm fine with that. Just stumbled across it, so put up a PR to not forget about it :)

Let me see how I can separate these changes from the others...

Heiko